staging/src/k8s.io/kube-aggregator/pkg/controllers/autoregister/autoregister_controller.go

```diff
@@ -153,8 +153,8 @@ func (c *autoRegisterController) Run(threadiness int, stopCh <-chan struct{}) {
 		}
 	}
 
-	// start up your worker threads based on threadiness.  Some controllers have multiple kinds of workers
-	for i := 0; i < threadiness; i++ {
+	// start up your worker threads based on workers.  Some controllers have multiple kinds of workers
+	for i := 0; i < workers; i++ {
```

Comment:
Just commenting out of curiosity, if the worker is already a thread and some controllers have multiple threads, why do we need to name it as a worker?

Reply:
The thread number of majority controllers are named `workers`, so just unify them for code consistency.